pep-0557.rst

```diff
@@ -745,7 +745,7 @@ to ``__post_init__`` where user code can use them as needed.
 
 The only real difference between alternate classmethod constructors
 and ``InitVar`` pseudo-fields is in regards to required non-field
-parameters during object creation.  With ``InitVar``s, using
+parameters during object creation.  With ``InitVar``\s, using
```

Comment:
I didn't know that worked! I tested locally and it looks good. Thanks.

I might make a similar edit elsewhere in the document the next time I take a pass over it.